### PR TITLE
Added a CloudWatch Events example to the Existing Event Formats secti…

### PR DESCRIPTION
**Pull request by timbray**
Added a CloudWatch Event example to the Existing Event Formats section. To the extent that AWS has a central Event format, this is it; the central event bus containing these things has become huge, as has the number of customers using the "CloudWatch Events" service to route them around to Lambdas and many other destinations.

Signed-off-by: Tim Bray <tbray@textuality.com> AKA <timbray@amazon.com>

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
@timbray any chance of you making the updates we agreed to on the call? I can merge it then

**Comment by timbray** [comment]
Got into a merge conflict in my own pull request, howzat?  Anyhow, I think this should work.

File: primer.md [hunk 1]
```diff
@@ -628,65 +628,27 @@ existing current event formats that are used in practice today was gathered.
...
-  ]
+  "version": "0",
+  "id": "6a7e8feb-b491-4cf7-a9f1-bf3703467718",
+  "detail-type": "EC2 Instance State-change Notification",
+  "source": "aws.ec2",
+  "account": "111122223333",
+  "time": "2017-12-22T18:43:48Z",
+  "region": "us-west-1",
+  "resources": [
+    "arn:aws:ec2:us-west-1:123456789012:instance/ i-1234567890abcdef0"
```
Comment (cneijenhuis) [inline primer.md]:
```suggestion
    "arn:aws:ec2:us-west-1:123456789012:instance/i-1234567890abcdef0"
```

File: primer.md [hunk 2]
```diff
@@ -628,65 +628,27 @@ existing current event formats that are used in practice today was gathered.
...
+  ],
+  "detail": {
+    "instance-id": " i-1234567890abcdef0",
```
Comment (cneijenhuis) [inline primer.md]:
```suggestion
    "instance-id": "i-1234567890abcdef0",
```

**Comment by timbray** [comment]
Argh, you're right.  Will fix up later today or early tomorrow.

**Comment by duglin** [comment]
@timbray any update?

**Comment by duglin** [comment]
Work done in: https://github.com/cloudevents/spec/pull/419

**Comment by duglin** [comment]
this was approved on last week's call - 4/11/19

**Pull request closed without merging**